fe/fe-core/src/main/java/com/starrocks/analysis/MultiRangePartitionDesc.java

```diff
@@ -127,6 +131,16 @@ public void setTimeUnit(String timeUnit) {
         List<SingleRangePartitionDesc> singleRangePartitionDescs = Lists.newArrayList();
         long currentLoopNum = 0;
         long maxAllowedLimit = Config.max_partitions_in_one_batch;
+
+        // By default, the dayOfWeek parameter are the same as those of dynamic partition.
+        // If user set dynamic_partition.start_day_of_week table properties
+        // it will follow this configuration to set day of week
+        int dayOfWeek = 1;
+        if (properties != null && properties.get(DynamicPartitionProperty.START_DAY_OF_WEEK) != null) {
+            dayOfWeek = Integer.parseInt(properties.get(DynamicPartitionProperty.START_DAY_OF_WEEK));
```

Comment:
When dayOfWeek is an invalid number, such as 9, can StarRocks handle it correctly?